add monad release docs outline (#717)
axelarnetwork/axelar-contract-deployments: releases/evm/2025-07-Monad-Transceiver-v1.1.0.md

```yaml
PRIVATE_KEY=xyz
ENV=xyz
CHAIN=monad
```

| Network              | `deployer address`                           |
|----------------------|----------------------------------------------|
| **Devnet-amplifier** | `0xba76c6980428A0b10CFC5d8ccb61949677A61233` |
| **Stagenet**         | `0xBeF25f4733b9d451072416360609e5A4c115293E` |
| **Testnet**          | `0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05` |
| **Mainnet**          | `0x6f24A47Fc8AE5441Eb47EFfC3665e70e69Ac3F05` |

### Create Transceiver Artifacts

Prerequisite: install Foundry (TODO: provide link)

<!-- or use Hardhat?? -->

1. Clone Transceiver repo: https://github.com/wormhole-foundation/example-wormhole-axelar-wsteth
2. `cd` into `example-wormhole-axelar-wsteth` and run `forge build` to create artifacts

- contracts will be compiled and ABIs generated in `out/` directory

### AxelarTrnasceiver Deployments

1. Set Environment Variables

```bash
ENV=xyz
CHAIN=xyz
```

1. Set paths to artifacts in `.env` in this directory:

```bash
TRANSCEIVER_STRUCTS_ADDRESS=0x..
GMP_MANAGER_ADDRESS=0x..
OPERATORS_ADDRESS=Ox..
PROXY_SALT="AXELAR_TRANSCEIVER v1.1.0"
THRESHOLD_VALUE=2
```

2. Deploy AxelarTransceiver contract

```bash

> ts-node evm/deploy-contract.js -c AxelarTransceiver --gmpManager $GMP_MANAGER_ADDRESS -m create2 --libraries '{"TransceiverStructs":"$TRANSCEIVER_STRUCTS_ADDRESS"}'

Reviewer: This needs to be a fully qualified library path. See the test cases written for the library linking process

## Monad Transceiver Deployment v1.1.0

|                | **Owner**                                                                     |
|----------------|-------------------------------------------------------------------------------|
| **Created By** | @kulikthebird <tomasz@interoplabs.io>, @blockchainguyy <ayush@interoplabs.io> |
| **Deployment** | @blockchainguyy <ayush@interoplabs.io>                                        |

| **Network**          | **Deployment Status** | **Date** |
|----------------------|-----------------------|----------|
| **Devnet Amplifier** | -                     | TBD      |
| **Stagenet**         | -                     | TBD      |
| **Testnet**          | -                     | TBD      |
| **Mainnet**          | -                     | TBD      |

## Background

> - This is the Monad Axelar/Wormhole Transceiver release. This release deploys AxelarTransceiver & ERC1967Proxy  contracts from the example-wormhole-axelar-wsteth [repo](https://github.com/wormhole-foundation/example-wormhole-axelar-wsteth).

Reviewer: ```suggestion
- This is the Monad Axelar/Wormhole Transceiver release. This release deploys [AxelarTransceiver](https://github.com/wormhole-foundation/example-wormhole-axelar-wsteth) and [OpenZeppelin ERC1967Proxy](https://github.com/smiley159/f2p/blob/master/node_modules/@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol) contracts.
```

Reviewer: I checked that now we use the wormhole's version of the ERC1967proxy contract - resolved

Reviewer: Let's decribe here why we used their fork of the OpenZeppelin contract here
